CONTRIBUTING.md

```diff
@@ -115,7 +114,7 @@ Run `npm start` to start the storybook server, then go to [http://localhost:9001
 
 If you want to add icons, they should first be approved by [Chris](https://skyscanner.slack.com/messages/D6BU4R0V8) as he "owns" Skyscanner icons.
```

Comment:
I know you haven't changed this line, but maybe we should remove this in the PR or change it to say message Backpack?

Reply:
Yeah lemme do that as a follow-up